### PR TITLE
Feature update comment

### LINKED ISSUE #181: API: Update comment
- User story #6

### PR DESCRIPTION
**Pull request by Sagi-Viktor**
- resolve #181

### CODE REVIEW

**Review by Zsezsu — APPROVED** [review]
nice job Viktor! :+1:

File: springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java [hunk 1]
```diff
@@ -22,18 +32,43 @@ class CommentServiceTest {
...
+
+    private CommentCreationDTO getCommentCreationDTO() {
+        return CommentCreationDTO.builder()
+                .postId(1L)
+                .postType(PostType.LEAGUE)
+                .timeStamp(date)
+                .user(AppUser.builder().id(1L).build())
+                .description("League has been created")
+                .build();
+    }
```
Comment (benedekhalaj) [inline springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java:53]: is this method used considering you implemented Mapstruct solution?

File: springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java [hunk 2]
```diff
@@ -44,4 +79,69 @@ public void deleteComment_commentDeletionById_commentNotInDB() {
...
+                        .build())
+                .build();
+        Comment expected = Comment.builder()
+                .id(1L)
+                .description("desc")
+                .timeStamp(date)
+                .user(AppUser.builder()
+                        .id(1L)
+                        .build())
+                .build();
```
Comment (benedekhalaj) [inline springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java:105]: I am not sure why you need to build these manually

**Review by benedekhalaj — CHANGES_REQUESTED** [review]

File: springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java (hunk 1 above)
Comment (Sagi-Viktor) [reply springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java:53]: Yeah I need to build a DTO to convert it

File: springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java (hunk 2 above)
Comment (Sagi-Viktor) [reply springboot/src/test/java/com/codecool/terraformingmarsforum/service/CommentServiceTest.java:105]: Same as previus answer

**Review by benedekhalaj — APPROVED** [review]

**Pull request merged**